content/en/docs/concepts/configuration/secret.md

```diff
@@ -271,6 +271,13 @@ However, using the builtin Secret type helps unify the formats of your credentia
 and the API server does verify if the required keys are provided in a Secret
 configuration.
 
+{{< caution >}}
+SSH private keys do not establish trusted communication between an SSH client and
+host server on their own. A secondary means of establishing trust is needed to
+mitigate "man in the middle" attacks, such as a `known_hosts` file added to a
```

Comment:
Hi @adambkaplan .
Is there a docs task or other content that explains how to add a `known_hosts` file [OpenSSH] to a `ConfigMap`?
https://en.wikibooks.org/wiki/OpenSSH/Client_Configuration_Files#/etc/ssh/ssh_known_hosts

Reply:
@kbhawkey Yes, we have a task on how to crate a ConfigMap from a file. Should I add a link here?

https://kubernetes.io/docs/tasks/configure-pod-container/configure-pod-configmap/#create-configmaps-from-files

Reply:
Thanks @adambkaplan . The change seems fine without the link.
The glossary entry for Secret links to this task:
```
Secrets are great for confidential data and [ConfigMaps](/docs/tasks/configure-pod-container/configure-pod-configmap/) for non-confidential data.
```
The ConfigMap concept page also links to the task page, https://deploy-preview-25498--kubernetes-io-master-staging.netlify.app/docs/concepts/configuration/configmap/#what-s-next.